src/main/java/org/terasology/metalrenegades/economy/events/TradeRequest.java

```diff
@@ -0,0 +1,24 @@
+package org.terasology.metalrenegades.economy.events;
+
+import org.terasology.entitySystem.entity.EntityRef;
+import org.terasology.entitySystem.event.Event;
+import org.terasology.metalrenegades.economy.ui.MarketItem;
+import org.terasology.network.ServerEvent;
+
+@ServerEvent
+public class TradeRequest implements Event {
+
+    public MarketItem pItem;
```

Comment:
Just add a little javadoc here explaining what these items are, and then we can merge :slightly_smiling_face:

Reply:
Javadoc has been added here along with other places where it was missing in this PR 👍